Userland/Libraries/LibGUI/AbstractSlider.h

```diff
@@ -38,6 +38,13 @@ class AbstractSlider : public Widget {
     void set_page_step(int page_step);
     void set_jump_to_cursor(bool b) { m_jump_to_cursor = b; }
 
+    void go_up(int delta) { set_value(value() + delta); }
+    void go_down(int delta) { set_value(value() - delta); }
+    void go_up_in_page_steps(int page_steps) { set_value(value() + page_step() * page_steps); }
+    void go_down_in_page_steps(int page_steps) { set_value(value() - page_step() * page_steps); }
+    void go_up_in_steps(int steps) { set_value(value() + step() * steps); }
+    void go_down_in_steps(int steps) { set_value(value() - step() * steps); }
```

Comment:
Would it make sense to use the naming convention increase/decrease instead of up/down? That would match with horizontal sliders as well, and also the min/max-getters.

Reply:
Yeah, I think it would be a better name

But I also think that using only 'increase/decrease' would make the code less clear about what is doing 🤔  I mean, I wouldn't know what is increasing/decreasing in this example

```cpp
void Scrollbar::scroll_by_page(const Gfx::IntPoint& click_position)
{
    float range_size = max() - min();
    float available = scrubbable_range_in_pixels();
    float rel_scrubber_size = unclamped_scrubber_size() / available;
    float page_increment = range_size * rel_scrubber_size;

    if (click_position.primary_offset_for_orientation(orientation()) < scrubber_rect().primary_offset_for_orientation(orientation()))
        decrease(page_increment);
    else
        increase(page_increment);
}
```

Maybe `increase_slider_by()` could work? What do you think?

Reply:
Let's not bikeshed here. `increase` is a good option, @elyse0 you choose something you think is good and push that. If people dislike it, they will comment or even change it later.